src/internal/cluster/injector.go

```diff
@@ -44,11 +42,9 @@ func (c *Cluster) StartInjectionMadness(tempPath types.TempPaths, injectorSeedTa
 	// Get all the images from the cluster
 	timeout := 5 * time.Minute
 	spinner.Updatef("Getting the list of existing cluster images (%s timeout)", timeout.String())
-	if images, err = c.Kube.GetAllImages(timeout); err != nil {
+	if images, err = c.GetAllImages(timeout); err != nil {
 		spinner.Fatalf(err, "Unable to generate a list of candidate images to perform the registry injection")
 	}
-	message.Debugf("Found %d images in the cluster", len(images))
-	message.Debugf("Images: %#v", images)
```

Comment:
I would argue these are useful debug statements - I have used them in the past.

Reply:
What specific information is helpful from this debug message? I would rather debug only the pertinent info (like `name:ns:uuid:image` etc...

Reply:
Actually looks like the same info is logged in the spinner in the `range images` function - knowing the images is useful though to figure out which ones it found / is trying to use since some images will not work such as those on nodes with taints.